src/resets.js

```diff
@@ -1,28 +1,25 @@
 // @flow
+import isSymbolsPage from './utils/isSymbolsPage';
 
 export const resetLayer = (container: Object) => {
-  const layers = container.children();
-  // Skip last child since it is the container itself
-  for (let l = 0; l < layers.count() - 1; l += 1) {
-    const layer = layers.objectAtIndex(l);
-    layer.removeFromParent();
-  }
```

Comment:
This was especially puzzling to me.

From a code standpoint, I don't understand "Skip last child since it is the container itself"... The values of `container.children()` are only the containers children. It would make no sense for `container.children()` to contain itself.

When manually running this piece of code, it did exactly what I expected it to... It left a child on the page. To be more clear. If I create a page, I insert and Artboard on that page, and I run `restLayer` on that page, the Artboard is not removed.

Reply:
Must be a user error of some sort, but upon further investigation this check does appear necessary when using `layer.children()`. However, we may be able to bypass this check by instead utilizing `layer.layers()` which appears to return only the direct descendants of the layer.